### PR TITLE
Add json output format

### PR DESCRIPTION
**Pull request by GokceGK**
Added json output format to the response of the following actions: 

- Argus instance create
- Config list
- DNS 
        - record set create
        - zone create
- LogMe 
        - credentials create
        - instance create
- MariaDB 
        - credentials create
        - instance create
        - instance update
- MongoDBFlex 
        - instance create
        - instance update
        - user create
        - user reset-password
- Object Storage 
        - bucket create
        - credentials group create
        - credentials create
- Opensearch 
        - credentials create
        - instance create
- PostgresFlex 
        - instance clone
        - instance create
        - instance update 
        - user create
        - user reset password
- Project create
- RabbitMQ 
        - credentials create
        - instance create
- Redis 
        - credentials create
        - instance create 
- Secrets-manager 
        - instance create
        - user create
- Service account
        - create
        - token create
- SKE 
        - cluster create
        - cluster update
        - kubeconfig create

### CODE REVIEW

File: internal/cmd/argus/instance/create/create.go [hunk 1]
```diff
@@ -102,12 +103,17 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				s.Stop()
 			}
 
-			operationState := "Created"
-			if model.Async {
-				operationState = "Triggered creation of"
+			switch model.OutputFormat {
```
Comment (vicentepinto98) [inline internal/cmd/argus/instance/create/create.go:106]: Move the `switch case` inside the `outputResult` function for uniformity with the other commands

File: internal/cmd/config/list/list.go [hunk 2]
```diff
@@ -35,56 +42,85 @@ func NewCmd(p *print.Printer) *cobra.Command {
...
+		},
+	}
+	return cmd
+}
 
-			table := tables.NewTable()
-			table.SetHeader("NAME", "VALUE")
-			for _, key := range configKeys {
-				value := configData[key]
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
```
Comment (vicentepinto98) [inline internal/cmd/config/list/list.go:62]: this function doesnt need to return error

File: internal/cmd/curl/curl.go [hunk 3]
```diff
@@ -212,7 +212,7 @@ func outputResponse(p *print.Printer, model *inputModel, resp *http.Response) er
 	}
 	respBody, err := io.ReadAll(resp.Body)
 	if err != nil {
-		return fmt.Errorf("read respose body: %w", err)
+		return fmt.Errorf("read response body: %w", err)
```
Comment (vicentepinto98) [inline internal/cmd/curl/curl.go:215]: :+1:

File: internal/cmd/argus/instance/create/create.go (hunk 1 above)
Comment (vicentepinto98) [reply internal/cmd/argus/instance/create/create.go:106]: same for all other commands

File: internal/cmd/opensearch/credentials/create/create.go [hunk 4]
```diff
@@ -121,3 +127,13 @@ func buildRequest(ctx context.Context, model *inputModel, apiClient *opensearch.
 	req := apiClient.CreateCredentials(ctx, model.ProjectId, model.InstanceId)
 	return req
 }
+
+func outputResult(p *print.Printer, resp *opensearch.CredentialsResponse) error {
+	details, err := json.MarshalIndent(resp, "", "  ")
+	if err != nil {
+		return fmt.Errorf("marshal OpenSearch credential: %w", err)
```
Comment (vicentepinto98) [inline internal/cmd/opensearch/credentials/create/create.go:134]:
```suggestion
		return fmt.Errorf("marshal OpenSearch credentials: %w", err)
```

File: internal/cmd/config/list/list.go (hunk 2 above)
Comment (GokceGK) [reply internal/cmd/config/list/list.go]: Removed the error from return values

File: internal/cmd/argus/instance/create/create.go (hunk 1 above)
Comment (GokceGK) [reply internal/cmd/argus/instance/create/create.go]: shifted condition handling in the outputresult

**Pull request merged**